docs/specifications/tests/DNSSEC-TP/dnssec12.md

```diff
@@ -0,0 +1,49 @@
+## DNSSEC12: Test for DNSSEC Algorithm Completeness
+
+### Test case identifier
+**DNSSEC12** Test for DNSSEC Algorithm Completeness
+
+### Objective
+
+The objectives for this Test Case has yet to be defined. This is a
+placeholder for a complete defintion of the Test Case. The Test Case
+is not yet implemented.
+
+Test for DNSSEC Algorithm Completeness (DS->DNSKEY->RRSIG)
+
+See issues [#588], [#528], [#529] and [#231].
+
+[#588]: https://github.com/dotse/zonemaster/issues/588
+[#528]: https://github.com/dotse/zonemaster/issues/528
+[#529]: https://github.com/dotse/zonemaster/issues/529
+[#231]: https://github.com/dotse/zonemaster/issues/231
+
```

Comment:
Will the issues documentation above removed finally?

Reply:
@sandoche2k, yes, the true specification DNSSEC12 will not contain those links. There here now to point out what to expect.